packages/api/test/e2e/promise-tx.spec.js

```diff
@@ -59,43 +73,13 @@ describe.skip('e2e transactions', () => {
     return api.tx.balances
       .transfer(keyring.eve.address(), 12345)
       .sign(keyring.dave, nonce)
-      .send(({ events, status }) => {
-        console.log('Transaction status:', status.type);
-
-        if (status.isFinalized) {
-          console.log('Completed at block hash', status.value.toHex());
-          console.log('Events:');
-
-          events.forEach(({ phase, event: { data, method, section } }) => {
-            console.log('\t', phase.toString(), `: ${section}.${method}`, data.toString());
-          });
-
-          if (events.length) {
-            done();
-          }
-        }
-      });
+      .send(loEvents(done));
```

Comment:
Should this be `logEvents`?

Reply:
Ok, so how the hell did I manage to screw up that line between testing and committing. Thanks!